descriptors/DeploymentDescriptor-template.json

```diff
@@ -0,0 +1,4 @@
+{
+  "srvcId": "${artifactId}-${version}",
+  "nodeId": "10.0.2.15"
```

Comment:
add plz run command like this for local or vagrant run
 "descriptor": {
    "exec": "java -Dport=%p -jar ../mod-email/target/mod-email-fat.jar -Dhttp.port=%p embed_postgres=true"
  }